### PR TITLE
Upgrade to Apache Yetus 0.12.0

### PR DESCRIPTION
**Pull request by aw-was-here**

### CODE REVIEW

File: api/proto/config/acipherinfo.proto [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright(c) 2017-2018 Zededa, Inc.
+// really? 2017-2018?
```
Comment (rvs) [inline api/proto/config/acipherinfo.proto:2]: Good point -- any chance you can fix those too as part of this?

Comment (aw-was-here) [reply api/proto/config/acipherinfo.proto:2]: I'd kind of like to avoid doing that in this patch in case this gets backed out for whatever reason.

Comment (eriknordmark) [reply api/proto/config/acipherinfo.proto:2]: @aw-was-here Is there a copyright check tool in Yetus?
Checking that 1) there is a Copyright and 2) that the year has been updated wouldn't be a bad idea.

Comment (aw-was-here) [reply api/proto/config/acipherinfo.proto]: It doesn't have that sort of test. Thinking about it, I suspect there are quite a few edge cases though. Imported code, when does a copyright get updated, how many different forms can a copyright notice take, etc.

**Comment by aw-was-here** [comment]
Now that 0.12.0 has officially been released...

**Comment by rvs** [comment]
LGTM
